website/content/docs/ecs/install-no-terraform.mdx

```diff
@@ -0,0 +1,492 @@
+| `name`        | string  | The container name should be `consul-ecs-mesh-init`.                                                                         |
+| `image`       | string  | The `consul-ecs` image. Use our public AWS registry, `public.ecr.aws/hashicorp/consul-ecs`, to avoid rate limits.            |
+| `mountPoints` | list    | The mounts two volumes so that mesh-init can invoke the `consul` binary to write bootstrap Envoy configuration.              |
+| `command`     | list    | Set the `["mesh-init"]` so that the container runs the `consul-ecs mesh-init` command.                                       |
+| `environment` | list    | This must include the `CONSUL_ECS_CONFIG_JSON` variable. See below for details.                                              |
+
+Additional configuration is passed to the `consul-ecs mesh-init` command in JSON format using the `CONSUL_ECS_CONFIG_JSON` environment variable.
+Here is the sample config from above, expanded to be readable:
+
+```json
+{
+  "bootstrapDir": "/consul",
+  "healthSyncContainers": [],
+  "proxy": {
+    "upstreams": [
+      {
+        "destinationName": "example-server-app",
+        "localBindPort": 1234
+      }
+    ]
+  },
+  "service": {
+    "checks": [],
+    "meta": {},
+    "name": "example-client-app",
+    "port": 9090,
+    "tags": []
+  }
+}
+```
+
+| Field name        | Type   | Description                                                                                                                    |
+| ----------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| `bootstrapDir`    | string | This is the path of a  shared volume the is mounted to other containers, where `mesh-init` will write out Envoy configuration. |
+| `proxy.upstreams` | list   | The upstream services that your application calls over the service mesh, if any.                                               |
+| `service.name`    | string | The name used to register this service into the Consul service catalog.                                                        |
+| `service.port`    | number | The port your application listens on. Set to `0` if your application does not listen on any port.                              |
+| `service.checks`  | list   | Consul [checks](/docs/discovery/checks) to include, to have Consul run health checks against your application. |
+
+See the [`consul-ecs JSON Schema`](https://github.com/hashicorp/consul-ecs/blob/main/config/schema.json) for a complete reference of fields.
+
+## `consul-ecs-health-sync` container
+
+Optionally, Consul ECS can sync health checks for this task into Consul checks.
+This allows you to configure a health check for your application in one place, and
+see a consistent health status in both ECS and Consul.
+
+First, you'll need to add an ECS health check command to the container definition for your application.
```

Comment:
This is telling users to add a check but really we're trying to say "if you _already_ have a check..."